mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestDerivates.java

```diff
@@ -394,29 +457,60 @@ boolean isDisplayEnabled() {
             return displayEnabled;
         }
 
-        @FormDataParam("display-enabled")
         @JsonProperty("display-enabled")
         public void setDisplayEnabled(boolean displayEnabled) {
             this.displayEnabled = displayEnabled;
         }
 
+        @FormParam("display-enabled")
+        @DefaultValue("true")
+        public void setDisplayEnabled(String displayEnabled) {
+            this.displayEnabled = Boolean.parseBoolean(displayEnabled);
+        }
+
         String getMainDoc() {
             return mainDoc;
         }
 
-        @FormDataParam("main-doc")
+        @FormParam("main-doc")
         @JsonProperty("main-doc")
         public void setMainDoc(String mainDoc) {
```

Comment:
param name should be "maindoc" as in derivate xml or xml of MCRMetaEnrichedLinkID